### PR TITLE
Auto-update Letsencrypt certificates for traefik

### PR DESCRIPTION
**Pull request by jeanpommier**
Use a sidecar container ([provided in traefik examples](https://gist.github.com/pyrou/4f555cd55677331c742742ee6007a73a#file-docker-compose-yml-L29)) to automatically update LE certificates

### CODE REVIEW

**Comment by fvanderbiest** [comment]
Do we need to host the files locally then ?

**Comment by jeanpommier** [comment]
They already are stored locally, in resources/ssl/. We might even remove the files from the git history, we won't need them anymore, I think

**Comment by fvanderbiest** [comment]
> We might even remove the files from the git history, we won't need them anymore

Please proceed.

**Comment by jeanpommier** [comment]
I believe the other files in resources/ssl (ca.* + readme instructions) are also deprecated ones, right @pmauduit ?

**Comment by pmauduit** [comment]
> right @pmauduit ?

yes

**Comment by jeanpommier** [comment]
OK. [Removed](https://github.com/georchestra/docker/pull/108/commits/d62ad51ba48789581215dbafb40caed473cbf6bd).

File: resources/ssl/README.md [hunk 1]
```diff
@@ -1,12 +1,6 @@
...
 
-This is how the certificate was created:
-```
-openssl req -new -newkey rsa:2048 -x509 -days 3650 -extensions v3_ca -keyout ca.pem -out ca.crt
-openssl rsa -in ca.pem -out ca.key
-openssl genrsa -out georchestra.mydomain.org.key 2048
-openssl req -key georchestra.mydomain.org.key -new -out georchestra.mydomain.org.csr
-openssl x509 -req -days 3650 -in georchestra.mydomain.org.csr -CA ca.crt -CAkey ca.key -CAcreateserial -out georchestra.mydomain.org.crt
-```
+It is empty (except this readme) when you clone it: the certificate's files are automatically downloaded and stored here by the reverse-proxy-https-helper container (see docker-compose.override.yml).
```
Comment (fvanderbiest) [inline resources/ssl/README.md]: `traefik-me-certificate-downloader`

Comment (jeanpommier) [reply resources/ssl/README.md]: oops

**Comment by fvanderbiest** [comment]
Nice addition, thanks Jean !

**Comment by fvanderbiest** [comment]
May need a backport ?

**Comment by jeanpommier** [comment]
Not sure it's worth the hassle...

**Comment by fvanderbiest** [comment]
Maybe we can take care of it later if we see a gain backporting it.

**Pull request merged**